theme/static/js/hs_file_browser.js

```diff
@@ -125,6 +126,12 @@ function updateSelectionMenuContext() {
         flagDisableDownload = true;
         flagDisableUnzip = true;
         flagDisableGetLink = true;
+        // Multiple folder deletion is not allowed for composite resource
+        // to avoid race condition that can occur with logical file objects especially
+        // if a folder has many files
+        if(resourceType === 'Composite Resource' && selected.length > 1){
```

Comment:
@mauriel133 If you think making this change ```selected.length > 2``` will solve this problem I will make that change. As to messages in terms of what is allowed or not for a specific resource type have we done this for any other resource types? I know for many resource types we don't allow zipping and creation of folder.